### ISSUE TITLE
Flow latency report should include flow names in leaf components

### ISSUE THREAD

**Issue by lwrage** [osate/osate2#2289]
## Summary

The report created by the flow latency analysis include the names of leaf components but not of the flow, which makes it hard to follow if there are multiple flows.

## Expected and Current Behavior

The flow source/path/sink name should be included in the report.

## Environment

* **OSATE Version**: 2.7.0
* **Operating System**: Linux

**Comment by AaronGreenhouse** [comment]
Added the flow spec name to be displayed next to the leaf component.
